runtime/src/chpl-privatization.c

```diff
@@ -57,8 +58,8 @@ void chpl_newPrivatizedClass(void* v, int64_t pid) {
       oldCap = chpl_capPrivateObjects;
       chpl_capPrivateObjects = 2*max(pid, oldCap);
 
-      tmp = chpl_mem_allocMany(chpl_capPrivateObjects, sizeof(void *),
-                               CHPL_RT_MD_COMM_PRV_OBJ_ARRAY, 0, 0);
+      tmp = chpl_mem_allocManyZero(chpl_capPrivateObjects, sizeof(void *),
```

Comment:
Note that for some memory layers `calloc` will cause pages to get touched, which can result in sub-optimal numa-affinity. I don't think that's an issue here, since this just stores the pointers to the classes, not the classes themselves, but I wanted to double check with you on that.